### PR TITLE
UI - fix warning

### LINKED ISSUE #3263: NewUI - confirming final tx does not return to main account view
Seems related to #3262.

While that one regards the app-initiated popup, this one is also reproducible within the extension "action view", by clicking the fox, and sending a tx. After clicking "confirm", a spinner is shown indefinitely.

A warning is visible in the UI:

```
bundle.js:121348 Warning: Failed prop type: Invalid prop `selected` of type `number` supplied to `AccountDropdowns`, expected `string`.
    in AccountDropdowns (created by Connect(AccountDropdowns))
    in Connect(AccountDropdowns) (created by App)
    in div (created by App)
    in div (created by App)
    in div (created by App)
    in div (created by App)
    in App (created by Connect(App))
    in Connect(App) (created by SelectedApp)
    in SelectedApp (created by Connect(SelectedApp))
    in Connect(SelectedApp) (created by Root)
```

### PR DESCRIPTION
**Pull request by kumavis**
Fixes #3263

found this orphaned commit on a branch without a PR
not sure if it actually fixes the issue or what, but didnt want it to get lost

cc @danfinlay

### CODE REVIEW

**Comment by danjm** [comment]
Let's hold off on merging this. It is possible that the observed bug resulted from a local build issue. Before we move further, I'd like to be able to consistently recreate the bug that motivated it (#3263). Then it would be good figure out what code change broke the relevant components. (They were behaving as expected during any QA efforts in recent weeks.)

Also, note that this code change happens in the `old-ui` directory, but the reported bug was for new ui.

**Comment by danfinlay** [comment]
> Also, note that this code change happens in the old-ui directory, but the reported bug was for new ui.

It's a problem on `uat`, not on the new UI. The new UI doesn't use this particular reference from what I can tell.

It also didn't seem that this fixed the bigger bug, it just fixed an occasional error/warning.

**Comment by danjm** [comment]
@danfinlay After further investigation, I say this PR is a good one to merge. I also added a commit that completes the intention of your commit while fixing https://github.com/MetaMask/metamask-extension/issues/3291

**Review by danjm — APPROVED** [review]

File: old-ui/app/app.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ App.prototype.renderAppBar = function () {
             style: {},
             enableAccountsSelector: true,
             identities: this.props.identities,
-            selected: this.props.currentView.context,
+            selected: this.props.selected,
```
Comment (danfinlay) [inline old-ui/app/app.js]: Wouldn't this just be `selected`, not `this.props.selected`?

Comment (danfinlay) [reply old-ui/app/app.js]: Oh nevermind, I see now.

**Comment by danfinlay** [comment]
Whoops, probably should've merged into `master`, eh??

**Pull request merged**